docs/src/pages/lab/slider/CustomIconSlider.js

```diff
@@ -37,7 +37,7 @@ class CustomIconSlider extends React.Component {
       <div className={classes.root}>
         <Typography id="slider-image">Image thumb</Typography>
         <Slider
-          classes={{ container: classes.slider }}
+          classes={{ root: classes.slider }}
```

Comment:
We might as well go back to `className` style
```suggestion
          className={classes.slider}
```